ewlegal-opinion.sty

```diff
@@ -26,32 +46,11 @@
 \titleformat{\subsubsection}{\normalfont}{\thesection}{1em}
   {\medskip\underline{#1}\medskip}
 
-%% Create intro and facts commands for supposed convience
-\newcommand{\intro}{\section{INTRODUCTION}}
-\newcommand{\facts}{\section{FACTS}}
-
-%% Page numbers
+%% Headers and Footers
 \fancyhf{} % clear all
 \renewcommand{\headrulewidth}{0pt} % remove the header rule
```

Comment:
Nit: looks like we are using the comment style `% Capitalized sentence without period` thruout the project. let's change `remove` to `Remove` here to be consistent.

Reply:
Ahh, I didn't catch those. They were copied over. Just for clarity my comment style (in latex) is `%% Capitol, sans period` for permanent comments, and `% anything` for impermanent comments and todos. If we want to change the style to something else, that's fine with me.

Reply:
Fixed.